CRM/ACL/BAO/ACLEntityRole.php

```diff
@@ -68,21 +61,30 @@ public static function retrieve(&$params, &$defaults) {
    *   true if we found and updated the object, else false
    */
   public static function setIsActive($id, $is_active) {
-    return CRM_Core_DAO::setFieldValue('CRM_ACL_DAO_EntityRole', $id, 'is_active', $is_active);
+    return CRM_Core_DAO::setFieldValue(__CLASS__, $id, 'is_active', $is_active);
   }
 
   /**
-   * Delete Entity Role records.
+   * Delete Dedupe Entity Role records.
    *
    * @param int $entityRoleId
    *   ID of the EntityRole record to be deleted.
    *
    */
   public static function del($entityRoleId) {
-    $entityDAO = new CRM_ACL_DAO_EntityRole();
```

Comment:
May be we should remove this function OR add a warning that says: 
```php
\CRM_Core_Error::deprecatedWarning(__CLASS__ . '::' . __FUNCTION__ . ' is deprecated, use ' . __CLASS__ . '::deleteRecord.' ); 
```

Reply:
Yeah - I think the api v3 would hit that? I'm happy to merge as is & we can look at that as a follow up